subsys/bootloader/Kconfig

```diff
@@ -154,6 +154,15 @@ config SB_BPROT_IN_DEBUG
 	depends on (HAS_HW_NRF_BPROT || HAS_HW_NRF_MPU)
 
 endif # SECURE_BOOT_DEBUG
+
+config SB_DISABLE_FLASH_PATCH
+	bool "Disable Cortex-M4 Flash Patch capabilities"
+	depends on SOC_NRF52840
+	help
+	  The flash patch can be used by malicious code to circumvent secure
```

Comment:
IMO the commit message should contain this information.

Reply:
Done